src/window.cpp

```diff
@@ -2,16 +2,16 @@
 #include "config.hpp"
 #include "drawable.hpp"
 #include "xcb_util.hpp"
+#include <cmath>
+#include <iostream>
 #include <vector>
 #include <xcb/xproto.h>
 
 constexpr bool k_horizontal_stacking = (dxp_width == 0);
 constexpr bool k_vertical_stacking = (dxp_height == 0);
 
-window::window (const int16_t x, ///< x coordinate of the top left corner
-                const int16_t y, ///< y coordinate of the top left corner
-                const std::vector<dxp_socket_desktop> &desktops)
-    : drawable (x, y, 0, 0) // Width and height will be calculated from config
+window::window (const std::vector<dxp_socket_desktop> &desktops)
+    : drawable (0, 0, 0, 0) // Width and height will be calculated from config
```

Comment:
Do we really need first two arguments then? I think there's a better way to code it.

Reply:
oh, the function requires it, i see. Still, those four zeros seem weird. Maybe set args by default?

Reply:
will do